RULES.md

```diff
@@ -1603,12 +1603,34 @@ Same as for [`FastTravelBetweenConsecutiveStopsNotice`](#FastTravelBetweenConsec
 
 </details>
 
-<a name="FeedExpirationDateNotice"/>
+<a name="FeedExpirationDate7DaysNotice"/>
 
-### feed_expiration_date
+### feed_expiration_date_6_days
 
-At any time, the published GTFS dataset should be valid for at least the next 7 days, and ideally for as long as the operator is confident that the schedule will continue to be operated.
-If possible, the GTFS dataset should cover at least the next 30 days of service.
+The dataset expiration date defined in `feed_info.txt` is in seven days or less. At any time, the published GTFS dataset should be valid for at least the next 7 days.
+
+### References
+* [General Publishing & General Practices](https://gtfs.org/best-practices/#dataset-publishing--general-practices)
+
+<details>
+#### Notice fields description
```

Comment:
this table doesn't render, I think we need a line break after "Notice fields description".